lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -119,9 +119,9 @@ defmodule ExUnit.Case do
   therefore reserved:
 
     * `:case`       - the test case module
```

Comment:
Reordered these to go from the outermost to the innermost concept, file > line > test.